frontend/src/metabase/query_builder/containers/QueryBuilder.jsx

```diff
@@ -139,7 +141,6 @@ const mapStateToProps = (state, props) => {
 
     isBookmarked: getIsBookmarked(state, props),
     isDirty: getIsDirty(state),
-    isNew: getIsNew(state),
```

Comment:
I've explained about the removal of `getIsNew` selector in this comment https://github.com/metabase/metabase/pull/31986/files?w=0#r1246535868. Basically, I don't think this has been used by anyone anymore.